WG_JS_CORE.md

```diff
@@ -97,34 +96,92 @@ Improving the docs, specs and having great tutorials and examples goes a long wa
   - Return values are documented, including object properties and types
 - Onboarding documentation exists to explain domain concepts and provide tutorials/examples for achieving common tasks
 
-### `M (P0)`: Revamped APIs 🔄 🤝 🧠
+### `M (P0)`: Revamped APIs 📦 🤝
 
 Revamping the APIs will make them easier to use and simpler to understand. Using modern syntax will make JS IPFS appealing to a wider segment of the JS community, aiding adoption and contributions. It also gives us scope to make performance optimisations and stability/debuggability improvements.
 
 - Async/await is used throughout the codebase and callback APIs are dropped
-- Support ArrayBuffer as an alternative to Buffer input argument
+- ArrayBuffer is supported as an alternative to Buffer input argument
```

Comment:
It'd be good to expand this to support more types so we can use [bl](https://www.npmjs.com/package/bl) to avoid Buffer creation and also some sort of Buffer pool one day.

Would we also not want to support typed arrays (Uint8Array and friends)?